src/frontend/next/src/components/Posts/Timeline.tsx

```diff
@@ -0,0 +1,57 @@
+import { Container, createStyles, Grid } from '@material-ui/core';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import PostComponent from '../Post';
+import { Post } from '../../interfaces/index';
```

Comment:
Drop the `/index`, you can import `../../interfaces` and it will resolve to `interfaces/index`